config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
```

Comment:
- [ ] When I try to look at the documentation, this happens:

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/617d14ca-9c0d-4327-853e-a7d431515a87)

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/d043df66-a762-4181-96bd-da4d117644d2)

Kindly, fix your swagger tests so your docs can be accessed. 👍